src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -531,6 +531,25 @@ function renderSubtreeIntoContainer(
   );
 
   if (__DEV__) {
+    if (container._reactRootContainer && container.nodeType !== COMMENT_NODE) {
+      const hostInstance = DOMRenderer.findHostInstance(
+        container._reactRootContainer.current,
+      );
```

Comment:
Ah, ok. Interesting.

What is the advantage of adding another method to `DOMRenderer` rather than setting a boolean flag on the `portalContainer` node?